datasets/batch_view/heavy_users/intro.md

```diff
@@ -0,0 +1,39 @@
+The `heavy_users` table provides information about whether a given client_id is
+considered a "heavy user" on each day (using submission date).
+
+#### Contents
+
+The `heavy_users` table contains one row per client-day, where day is
+submission_date. A client has a row for a specific submission_date if
+they were active at all in the 28 day window ending on that submission_date.
+
+A user is a "heavy user" as of day N if, for the 28 day period ending
+on day N, the sum of their active_ticks is in the 90th percentile (or
+above) of all clients during that period. For more analysis on this,
+and a discussion of new profiles, see
+[this link](https://metrics.mozilla.com/protected/sguha/heavy/heavycutoffs5.html).
+
+#### Background and Caveats
```

Comment:
Another part to possibly add here is that there is no "fixing" of new profiles; days before they were created are just counted as 0 `active_ticks`.

Reply:
This is also the reason I included `profile_creation_date`, so people can leave out, adjust, or leave unchanged new profiles when doing analysis.